code/modules/research/designs/engineering.dm

```diff
@@ -162,3 +162,13 @@
 	materials = list(MAT_IRON = 500, MAT_GLASS = 100)
 	build_path = /obj/item/device/t_scanner/advanced
 	category = "Engineering"
+
+/datum/design/experimental_welder
```

Comment:
This is exceptionally cheap for something that basically invalidates fuel tanks.
It's also content creep.